src/resources/app-en.json

```diff
@@ -592,9 +596,11 @@
   "maxCharacters": "Max characters",
   "audioPlayerDescription": "The respondent will listen to an audio stimulus, no response is required.",
   "mediaTranscript": "Media transcript",
-  "uploadAudioRestrictions": "Please upload WAV or MP3 less than 8MB.",
+  "uploadAudioRestrictions": "Please upload WAV or MP3 less than {{size}}.",
   "uploadAudioDropText": "Drop Audio here or click to browse",
   "uploadAudioDescription": "Please Upload WAV or MP3",
-  "audioExceedSize": "Audio is more than 8Mb",
-  "audioWrongFormat": "Audio file has wrong format"
+  "audioExceedSize": "Audio is more than {{size}}",
+  "audioWrongFormat": "Audio file has wrong format",
+  "drawingExample": "Drawing Example",
```

Comment:
as I see in the mock up the text is "Image for drawing"
![image](https://user-images.githubusercontent.com/116339462/226295198-53fbfc73-7c29-4e3c-bafe-ccc3381cecbd.png)